### ISSUE TITLE
[Feature request] Customize Colors for the Document Scanner Activity

### ISSUE THREAD

**Issue by bensek** [googlesamples/mlkit#847]
**What's your feature request? Please describe.**
Currently, there is no way to change the colors of the button to use primary color of our app. Also, the background color of the scanner activity is always black. It would be great if this changes best on the dark and light theme of the app. 

**Mobile environment**
Android

**Comment by mebjas** [comment]
Thanks for the feature request. Our team is triaging this request and will get back.

Optional, It would be helpful if you could share more about the client app and how it's using doc scanner for us to understand the use-case and prioritize this better.

**Comment by bensek** [comment]
We have been using ScanBot SDK for a while in our app, and it offered us a lot of customization options. We would love to have the same ability with MLKit document scanner especially the changing the button colors.

**Comment by chattansgpt** [comment]
That's right. It's high time that the ML Kit doc scanner, QR scanner and Camera take picture intent need an option to customize the theme color based on the app. My app is all light color themed and it looks wierd when it goes to a camera screen which looks nowhere related to the app
